deps/build.jl

```diff
@@ -125,11 +138,55 @@ copy_config("ijuliafavicon.ico",
 # custom.js can contain custom js login that will be loaded
 # with the notebook to add info and/or monkey-patch some javascript
 # -- e.g. we use it to add .ipynb metadata that this is a Julia notebook
-copy_config("custom.js", joinpath(juliaprof, "static", "custom"))
+if ipyvers <= v"3.0-"
+  copy_config("custom.js", joinpath(juliaprof, "static", "custom"))
+else
+  # TODO
+  ## upgrade custom.js because old version can prevent notebook
+  ## from loading.
+  ## todo: maybe do not copy if don't exist.
+  ## todo: maybe remove if user custom.js is identical to the one
+  ## shiped with IJulia ?
+  copy_config("custom.js", joinpath(juliaprof, "static", "custom"))
+end
 
 # julia.js implements a CodeMirror mode for Julia syntax highlighting in the notebook.
 # Eventually this will ship with CodeMirror and hence IPython, but for now we manually bundle it.
 
-copy_config("julia.js", joinpath(juliaprof, "static", "components", "codemirror", "mode", "julia"))
+if ipyvers <= v"3.0-"
+    copy_config("julia.js", joinpath(juliaprof, "static", "components", "codemirror", "mode", "julia"))
```

Comment:
Can't we do something like what you did with `custom.js` to make `julia.js` work with IPython 3?

Suppose someone has IPython 2.0 installed and installs IJulia (hence gets `julia.js`) then upgrades to IPython 3.  Won't the `julia.js` file lying around cause things to break?